### ISSUE TITLE
Error when deploy with replset enabled

### ISSUE THREAD

**Issue by tuchinsky** [stone-payments/ansible-mongodb#8]
I have 2 servers with ubuntu 16.04.4 LTS

my ansible playbook:

```
- hosts: mongo

  roles:
  - stone-payments.mongodb
  vars:
  - mongodb_conf_bindIp: "0.0.0.0"
  - mongodb_replSet_enabled: true
  - mongodb_replSet_name: "someReplicaSetName"
  - mongodb_replSet_master: "192.168.1.65"
  - mongodb_replSet_isMaster: "{{ true if mongodb_replSet_master in ansible_all_ipv4_addresses else false }}"
```

When I run this playbookt, I get an error:

```
TASK [stone-payments.mongodb : connect to master] *******************************************************************************************************************************************************************************************
fatal: [ubuntu2]: FAILED! => {"changed": false, "module_stderr": "Shared connection to 192.168.1.66 closed.\r\n", "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_Fq6YXJ/ansible_module_mongodb_replica_set.py\", line 447, in <module>\r\n    main()\r\n  File \"/tmp/ansible_Fq6YXJ/ansible_module_mongodb_replica_set.py\", line 442, in main\r\n    result['changed'] = rs_alter(client, member, state, 0)\r\n  File \"/tmp/ansible_Fq6YXJ/ansible_module_mongodb_replica_set.py\", line 343, in rs_alter\r\n    current_member = rs_get_member(rs_config, member['host'])\r\n  File \"/tmp/ansible_Fq6YXJ/ansible_module_mongodb_replica_set.py\", line 289, in rs_get_member\r\n    a = filter(lambda x: x['host'] == member, rs_config['members'])\r\nTypeError: 'NoneType' object has no attribute '__getitem__'\r\n", "msg": "MODULE FAILURE", "rc": 0}
```

**Comment by bcdonadio** [comment]
I believe you're having an issue by not having a password and/or replication key definition. I'm building a test case for the replica set scenario that should help me further debug this issue and the fix it.

**Comment by bcdonadio** [comment]
Hi. I found the issue. It was a mixup between user authentication and replica authentication that I made some time ago.

It was fixed in #16 and a test-case was built for it.

**Comment by tuchinsky** [comment]
thanks!
